device/arista/x86_64-arista_7280cr3_32d4/platform_asic

```diff
@@ -0,0 +1 @@
+broadcom
```

Comment:
this is available for sonic-broadcom-dnx.bin not in sonic-broadcom.bin

can you differ?

Reply:
Technically they are using the same CONFIGURED_PLATFORM. So I cannot differentiate them by this dimention. We could improve in future.

Reply:
this is a must fix

Reply:
please sync with judy

Reply:
I fixed the files under dnx platforms. I decided not reduce package content and keep the build/installation behavior as before.